### PR TITLE
[fix bug 1442129] Newsletter welcome program opt-out confirmation page

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Adds an opt-out confirmation page for the Mozilla welcome email program.

~~Do not merge until we get some design eye balls on this.~~

## Issue / Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1442129

## Testing
Demo: https://bedrock-demo-agibson.us-west.moz.works/en-US/newsletter/opt-out-confirmation/

- [ ] Test email recovery form works as expected
- [ ] Ensure no regressions in standard `/newsletter/recovery/` page.

### CODE REVIEW

**Comment by alexgibson** [comment]
OK, this is now ready for code review.

File: bedrock/newsletter/views.py [hunk 1]
```diff
@@ -596,12 +596,12 @@ def recovery(request):
     else:
         form = EmailForm()
 
-    return l10n_utils.render(
-        request,
-        "newsletter/recovery.html",
-        {
-            'form': form,
-        })
+    if '/newsletter/opt-out-confirmation/' in request.get_full_path():
```
Comment (jpetto) [inline bedrock/newsletter/views.py]: Maybe add a comment here detailing the difference in views?

File: bedrock/newsletter/templates/newsletter/opt-out-confirmation.html [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+              <li><a href="https://blog.mozilla.org/">{{ _('Check out our blogs') }}</a></li>
+              <li><a href="https://support.mozilla.org/">{{ _('Get help') }}</a></li>
+              <li><a href="{{ url('newsletter.firefox') }}">{{ _('Subscribe to occasional newsletter updates from Firefox') }}</a></li>
+            </ul>
+
+            <ul class="social-links">
+              <li><a class="instagram" href="https://www.instagram.com/mozillagram/" data-link-type="footer" data-link-name="Instagram (@mozillagram)">{{ _('Instagram') }}<span> (@mozillagram)</span></a></li>
+              <li><a class="youtube" href="https://www.youtube.com/firefoxchannel" data-link-type="footer" data-link-name="YouTube (firefoxchannel)">{{ _('YouTube') }}<span> (firefoxchannel)</span></a></li>
+              <li><a class="facebook" href="https://www.facebook.com/Firefox" data-link-type="footer" data-link-name="Facebook (Firefox)">{{ _('Facebook') }}<span> (Firefox)</span></a></li>
+              <li><a class="twitter" href="{{ firefox_twitter_url() }}" data-link-type="footer" data-link-name="Twitter (@firefox)">{{ _('Twitter') }}<span> (@firefox)</span></a></li>
```
Comment (jpetto) [inline bedrock/newsletter/templates/newsletter/opt-out-confirmation.html]: These are all invisible. 👻

File: bedrock/newsletter/templates/newsletter/opt-out-confirmation.html [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+{% endblock %}
+
+{% block content %}
+<main>
+  <div class="content">
+    <div class="inner-content">
+      <header>
+          <h1>{{ self.page_title() }}</h1>
+          <p class="tagline">{{ self.page_desc() }}</p>
+        </header>
```
Comment (jpetto) [inline bedrock/newsletter/templates/newsletter/opt-out-confirmation.html]: Nit: indentation.

**Comment by alexgibson** [comment]
@jpetto thanks for the review - I added a commit with some fixes.

**Pull request merged**